README.md

```diff
@@ -50,74 +57,80 @@ Docker Compose will download the required Docker images, before starting the dem
 3. Navigate to the [MLT dashboard](http://localhost:3000/d/4VSk5Lank/mlt-dashboard?orgId=1&refresh=5s).
 4. Explore the data sources using the [Grafana Explorer](http://localhost:3000/explore?orgId=1&left=%7B%22datasource%22:%22Mimir%22,%22queries%22:%5B%7B%22refId%22:%22A%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D).
 
+The [pre-provisioned dashboard](grafana/definitions/mlt.json) demonstrates a [RED (Rate, Error, Duration)](https://grafana.com/blog/2018/08/02/the-red-method-how-to-instrument-your-services/) overview of the microservice application, where almost all metrics are being generated via trace spans. There is also an example of logging. [Data links](https://grafana.com/docs/grafana/latest/panels-visualizations/configure-data-links/), [exemplars](https://grafana.com/docs/grafana-cloud/data-configuration/traces/exemplars/) and logs are utilised to allow jumping from the dashboard to a Grafana Explore page to observe traces, metrics and logs in more detail.
+
 The following is a brief explanation of each of the most important provided components.
 
 ## Grafana
 
-Grafana is a visualisation tool that allows the creation of dashboards from various data sources. More information can be found [here](https://grafana.com/docs/grafana/latest/).
+Grafana is a multi-platform open source analytics and interactive visualisation web application. For more details about Grafana, please see the [documentation](https://grafana.com/docs/grafana/latest/).
 
-The Grafana instance is described in the `grafana` section of the `docker-compose.yml` manifest.
+The Grafana service is described in the `grafana` section of the [`docker-compose.yml`](docker-compose.yml) manifest.
 
-It:
-* Mounts two repository directories to provide pre-provisioned data sources for data.
-* A pre-provisioned dashboard for correlating metrics, logs and traces.
-  * This dashboard uses metrics from span traces to provide RED (Rate/Error/Duration) signals.
+This definition:
```

Comment:
What do you mean by this definition? Is it the grafana definition in the docker-compose.yaml file?

Reply:
I mean the docker-compose file. I'll make it clearer.

Reply:
Done!